scripts/apply_freezer_location.py

```diff
@@ -0,0 +1,35 @@
+#!/opt/gls/clarity/users/glsai/Applications/clarity_scripts/UserPreparedLibrary20180523/bin/python3.4
```

Comment:
This should be `#!/usr/bin/env python` and will be expanded to the python's full path once deployed